docs/04-guida-sviluppo/index.md

```diff
@@ -40,14 +50,74 @@ Quelle supportate sono:
 * Schemi dati in formato [OAS3] (OpenAPI Specifications versione 3).
   Future versioni del catalogo possono supportare altre risorse semantiche ed altri formati.
 
+## File richiesti e layout del repository
+
+Il repository DEVE contenere i seguenti file:
+
+- ndc-config.yaml: referenziando la posizione delle risorse semantiche
+  ed ulteriori informazioni necessarie
+  alla pubblicazione su NDC;
+- publiccode.yaml: contenente tutte le informazioni richieste dal
+  [Catalogo del Riuso](https://developers.italia.it/it/software).
+
+Un repository è a tutti gli effetti un oggetto pubblico indicizzato dal Catalogo del Riuso,
+e DEVE contenere un file [publiccode.yml conforme alle relative Linee Guida](https://docs.italia.it/italia/developers-italia/publiccodeyml)
+col riferimento al [codice IPA](https://www.indicepa.gov.it/) dell'ente che gestisce il repository.
+
+Queste informazioni verranno utilizzate anche per la continuità operativa del NDC.
+
+```yaml
+...
+maintenance:
+  contacts:
+email: info@teamdigitale.governo.it
+name: Dipartimento per la Trasformazione Digitale
+it:
+  riuso:
+    codiceIPA: pcm
+```
+
+Tutte le risorse fornite DEVONO risiedere all'interno della cartella `assets/`
+referenziato in ndc-config.yaml.
+Le risorse al di fuori du `assets/` non saranno elaborate.
+
+Ogni tipo di asset (ontologie, vocabolari controllati, schemi)
+DEVE risiedere nella sua cartella specifica con un nome predefinito,
+referenziato in ndc-config.yaml.
```

Comment:
Il fatto di mettere i nomi nelle directory nell' `ndc-config.yaml` serviva a gestire la legacy.
Verificate se a regime ha senso, poiché questa feature potrebbe essere abusata.

Reply:
@Clou-dia

Reply:
Tracked in #19